src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/base/_fonts.scss

```diff
@@ -39,5 +39,5 @@
 }
 
 // Font Awesome config
-$fa-font-path: '../vendor/font-awesome/fonts';
+$fa-font-path: '../fonts';
 
```

Comment:
Mja, bij fork uploaden we die node_modules als ik me niet vergis, want beetje dom is.
Heb je een voorkeur? Kopieren uit de node_modules lijkt me beste aangezien we dan steeds met de juiste versie zitten, en we niet dubbel moeten zetten. Akkoord?